README.md

```diff
@@ -209,7 +209,36 @@ always read lower case. We can do:
 util.Try[String] = Success(foobar)
 ```
 
+## Handling missing keys
+
+The default behavior of `ConfigConverts` that are derived in PureConfig is to
```

Comment:
Nitpick: I'd move the "s" outside the backticks, i.e. `ConfigConvert`s.
Ditto below in a few places.
Feel free to ignore.